spec/spec_helper.rb

```diff
@@ -1,7 +1,13 @@
 require 'simplecov'
 SimpleCov.start
 require 'pry'
-require './lib/blackjack'
```

Comment:
By requiring this, we were starting a game, which asks the user for input. That's no good in test mode, so I changed spec helper to require the files individually. It would be even better if each spec only required the files it needs, but this works fine.